protokt-runtime/src/commonMain/kotlin/protokt/v1/Bytes.kt

```diff
@@ -38,12 +44,14 @@ class Bytes(internal val value: ByteArray) {
     companion object {
         private val EMPTY = Bytes(ByteArray(0))
 
+        @JvmStatic
```

Comment:
I could have sworn I tried to put JvmStatic in common code before and the compiler threw up - now that it's working, I'm going to consider adding it to the generated code in the right spots before going 1.0, if there are any right spots. Maybe the constructor functions if I put them in companion objects.